src/Cosi-BLS/cosi-construction.lisp

```diff
@@ -34,12 +34,14 @@ THE SOFTWARE.
 ;; --------------------------------------------------------------------
 ;; Physical network
 
-(defvar *local-nodes*  '(("Arroyo.local"    . "127.0.0.1" #|"10.0.1.33"|#)
+(defvar *local-nodes*  '(("Arroyo.local"    . "127.0.0.1")))
```

Comment:
@dbmcclain Let's change `Arroyo.local`  to `localhost` which would be meaningful to others than just your network.